backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/statement/dto/StatementAnswerDetailsDto.java

```diff
@@ -0,0 +1,26 @@
+package pt.ulisboa.tecnico.socialsoftware.tutor.statement.dto;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import pt.ulisboa.tecnico.socialsoftware.tutor.answer.domain.AnswerDetails;
+import pt.ulisboa.tecnico.socialsoftware.tutor.answer.domain.QuestionAnswer;
+import pt.ulisboa.tecnico.socialsoftware.tutor.question.Updator;
+import pt.ulisboa.tecnico.socialsoftware.tutor.statement.domain.QuestionAnswerItem;
+
+import static pt.ulisboa.tecnico.socialsoftware.tutor.question.domain.Question.QuestionTypes.MULTIPLE_CHOICE_QUESTION;
+
+@JsonTypeInfo(
+        use = JsonTypeInfo.Id.NAME,
+        include = JsonTypeInfo.As.PROPERTY,
+        defaultImpl = MultipleChoiceStatementAnswerDetailsDto.class,
+        property = "type")
+@JsonSubTypes({
+        @JsonSubTypes.Type(value = MultipleChoiceStatementAnswerDetailsDto.class, name = MULTIPLE_CHOICE_QUESTION)
+})
+public abstract class StatementAnswerDetailsDto implements Updator {
+    public abstract AnswerDetails getAnswerDetails(QuestionAnswer questionAnswer);
+
+    public abstract boolean noAnswer();
+
+    public abstract QuestionAnswerItem getQuestionAnswerItem(String username, int quizId, StatementAnswerDto statementAnswerDto);
```

Comment:
Why is this method necessary in this context? QuestionAnswerItem is self-contained.